### PR TITLE
CP-46076 for XS9: To remove fcoe-utils, remove FCOE capabilty with LLDPAD tools from bugtool?

### PR DESCRIPTION
**Pull request by DeliZhangX**
The solution is just reverting those commits.

### CODE REVIEW

**Comment by bernhardkaindl** [comment]
@DeliZhangX: I added the explanation to CP-46076 why this change is not needed to remove the dependency on `fcoe-utils` for XS9.

In fact, two of the tools that would no longer be called if this would be merged are not even in fcoe-utils.
```py
rpm -qf /usr/sbin/lldptool /usr/sbin/dcbtool
lldpad-1.0.1-10.xs8.x86_64
lldpad-1.0.1-10.xs8.x86_64
```
As I said in my explanation in CP-46076, xen-bugtool is happy to work even if a few tools and files that it is asked to collect, it just ignores those files and tools that are not found.

Therefore, this PR is not needed XS9 wants to make any XS9-specific changes, they would have to go into an XS9 branch, not the master branch which is currently in use for the XS8 product and is good for backports to Yangze as well.

As I explained in my comment to CP-46076, the only change that you need to make to remove `fcoe-utils` from XS9 is to disable the Requires of it the `xenserver-status-report.spec` at line 32:
```diff
+ %if 0%{?xenserver} < 9
  Requires:       fcoe-utils
+ %endif
```

File: xen-bugtool [hunk 1]
```diff
@@ -268,7 +266,6 @@ BIOSDEVNAME = 'biosdevname'
 BRCTL = 'brctl'
 CHKCONFIG = 'chkconfig'
 CHRONYC = 'chronyc'
-DCBTOOL = 'dcbtool'
```
Comment (bernhardkaindl) [inline xen-bugtool:271]: Only as a side note, for reference (obsolete) as this PR is not needed:

This package is not a file from `fcoe-utils`:
```py
rpm -qf /usr/sbin/dcbtool
lldpad-1.0.1-10.xs8.x86_64
```

File: xen-bugtool [hunk 2]
```diff
@@ -286,7 +282,6 @@ IPTABLES = 'iptables'
 ISCSIADM = 'iscsiadm'
 KPATCH = 'kpatch'
 LIST_DOMAINS = 'list_domains'
-LLDPTOOL = 'lldptool'
```
Comment (bernhardkaindl) [inline xen-bugtool:289]: Only as a side note, for reference (obsolete) as this PR is not needed:

This package is not a file from `fcoe-utils`:
```py
rpm -qf /usr/sbin/lldptool
lldpad-1.0.1-10.xs8.x86_64
```

**Comment by bernhardkaindl** [comment]
Because I added comments on other files, just to re-iterate: 

The only needed change for disabling the dependency of `xenserver-status-report.rpm` on `fcoe-utils` is to remove the `Requires:` for `fcoe-utils` for XS9 at line 32 in `xenserver-status-report.spec`:
```diff
+ %if 0%{?xenserver} < 9
  Requires:       fcoe-utils
+ %endif
```

**Comment by DeliZhangX** [comment]
> Also, they would not target master, which is currently in use for the XS8 product and is good for backports to the Yangtze as well (if at all, into a new release/xs9 branch.

Thanks for you comments.
May I ask when you will merge master and xs8 and yangtze to single one branch?

**Pull request closed without merging**